models/UserModel.js

```diff
@@ -43,14 +29,28 @@ const verifyPassword = (plainPassword, hashedPassword) =>
 const validate = (data) =>
   Joi.object({
     username: Joi.string().max(255).required(),
-    password: Joi.string().min(8).max(100).required(),
+    password: Joi.string().min(1).max(100).required(),
```

Comment:
One seems really short for a minimum password length 😉 

```suggestion
    password: Joi.string().min(4).max(100).required(),
```